db/sql_changeset_schema

```diff
@@ -39,6 +39,36 @@ CREATE INDEX user_account_devices_userid_idx on user_account_devices(userid);
 CREATE INDEX user_files_userid_idx on user_files(userid);
 CREATE INDEX user_files_deviceid_idx on user_files(deviceid);
 
+-- share file
+CREATE TABLE user_share_files (
+    id SERIAL PRIMARY KEY,
+    ownerid INTEGER NOT NULL,
+    uuid UUID UNIQUE,

```

Comment:
It's better to use VARCHAR not UUID. Some day we might decide to use some shorter code such as set of [a-zA-Z0-9]. VARCHAR would be OK for both UUID and String.